OPHD/StructureManager.h

```diff
@@ -69,6 +93,8 @@ class StructureManager
 		{
 			throw std::runtime_error("Structure::Attach() was called on a Structure that already had the component!");
 		}
+#else
+		UNUSED(success);
 #endif
```

Comment:
If the #define guard was removed the UNUSED macro could be avoided.

Reply:
sure, I can throw in release builds if you'd prefer that

Reply:
Throw in release is preferred.